pos_accounting/src/lib.rs

```diff
@@ -1,3 +1,24 @@
+use common::primitives::H256;
+use serialization::{Decode, Encode};
+
 pub mod error;
 pub mod pool;
 pub mod storage;
+
+#[derive(Clone, Copy, Debug, Encode, Decode, Default, Eq, PartialEq, Ord, PartialOrd)]
+pub struct PoolId(H256);
```

Comment:
not sure whether I should move these new types to `id.rs` or leave here

Reply:
Perhaps the better way to do this is defining an empty struct, like Pool, and doing `Id<Pool>`. Then we can later decide what we want to attach this to. 

The problem with the current format is that you have to redefine everything we defined for `Id<>`